manifests/init.pp

```diff
@@ -169,4 +170,9 @@
     type => 'string',
     data => $target_group,
   }
+
+  wsus_client::setting{ "${_au_base}\\AlwaysAutoRebootAtScheduledTime":
+    data           => $always_auto_reboot_at_scheduled_time,
+    validate_range => [0,1],
```

Comment:
Because this is a 0 or 1 value, we probably want to treat it / expose it as a boolean value to the end user.

See https://technet.microsoft.com/en-us/library/cc708449(v=ws.10).aspx for other 0 / 1 type values.